mmtrack/datasets/dancetrack_dataset.py

```diff
@@ -0,0 +1,486 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import os
+import os.path as osp
+import tempfile
+
+import mmcv
+import motmetrics as mm
+import numpy as np
+from mmcv.utils import print_log
+from mmdet.core import eval_map
+from mmdet.datasets import DATASETS
+
+from mmtrack.core import interpolate_tracks, results2outs
+from .mot_challenge_dataset import MOTChallengeDataset
+
+try:
+    import trackeval
+except ImportError:
+    trackeval = None
+
+
+@DATASETS.register_module()
+class DanceTrackDataset(MOTChallengeDataset):
```

Comment:
Please remove the duplicated codes since we have already inherited from `MOTChallengeDataset`.

Reply:
Removed in the new commit.